testnet/stacks-node/src/nakamoto_node/sign_coordinator.rs

```diff
@@ -407,6 +408,7 @@ impl SignCoordinator {
                 .filter_map(|msg| match msg {
                     SignerMessage::DkgResults { .. }
                     | SignerMessage::BlockResponse(_)
+                    | SignerMessage::EncryptedSignerState { .. }
```

Comment:
Just curious, why not `SignerMessage::EncryptedSignerState(_)`? Just a preference thing?

Reply:
Not a preference, just an artifact that I missed from when I developed this because I started it out with this as a struct variant. Will update, thanks for noting :pray:

Reply:
Updated.